packages/mongo-nav/src/user-menu/UserMenu.tsx

```diff
@@ -380,18 +400,20 @@ function UserMenu({
           <SubMenu
             {...subMenuContainer.prop}
             {...sharedProps}
+            data-testid="user-submenu-cloud"
             active={isCloud}
             disabled={!account}
             href={hosts.cloud}
+            title={isGovernment ? 'Cloud for Government' : 'Cloud'}
+            glyph={<CloudIcon {...ariaHiddenProps} />}
+            onClick={onElementClick(NavElement.UserMenuCloud, onClick)}
```

Comment:
man I hope `onElementClick`/`NavElement` go away some day. code that primarily exists for tests is 👃

Reply:
onElementClick is primarily for analytics, not tests.